src/pages/PassportRedemption/TrackScreen.tsx

```diff
@@ -156,40 +168,42 @@ const Track = ({ setCurrentScreenView }: Props) => {
             )}
           </Column>
 
-          <Row>
-            <Label htmlFor="instagram-handle">
-              Instagram Handle (for Digital Giveaway)
-            </Label>
-            <SupporterTooltip
-              title={
-                <React.Fragment>
-                  <ToolTipTable>
-                    <tbody>
-                      <tr>
-                        To be entered into our weekly Digital Giveaways, visit 3
-                        merchants and post and tag <b>@sendchinatownlove</b>{' '}
-                        with your food crawl pictures on Instagram. Enter your
-                        Instagram handle so we can track your entries.
-                      </tr>
-                    </tbody>
-                  </ToolTipTable>
-                </React.Fragment>
-              }
-              enterTouchDelay={10}
-              placement="left"
-            >
-              <div>
-                <img src={CrawlInfoIcon} alt="instagram-crawl-info" />
-              </div>
-            </SupporterTooltip>
-          </Row>
-          <InputField
-            name="instagram-handle"
-            type="text"
-            onChange={(e) => setinstagramHandle(e.target.value)}
-            value={instagramHandle}
-            placeholder="@"
-          />
+          <Column>
+            <Row className="row-no-margin">
+              <Label htmlFor="instagram-handle">
+                Instagram Handle (for Digital Giveaway)
+              </Label>
+              <SupporterTooltip
+                title={
+                  <React.Fragment>
+                    <ToolTipTable>
+                      <tbody>
+                        <tr>
+                          To be entered into our weekly Digital Giveaways, visit 3
+                          merchants and post and tag <b>@sendchinatownlove</b>
+                          &nbsp;with your food crawl pictures on Instagram. Enter your
+                          Instagram handle so we can track your entries.
+                        </tr>
+                      </tbody>
+                    </ToolTipTable>
+                  </React.Fragment>
+                }
+                enterTouchDelay={10}
+                placement="left"
+              >
+                <div>
+                  <img src={CrawlInfoIcon} alt="instagram-crawl-info" />
+                </div>
+              </SupporterTooltip>
+            </Row>
+            <InputField
+              name="instagram-handle"
+              type="text"
+              onChange={(e) => setinstagramHandle(e.target.value)}
+              value={instagramHandle}
+              placeholder="@"
```

Comment:
Are expecting users to type the `@` in themselves? I'm wondering if we should take the `$` [approach we did on the voucher page](https://github.com/sendchinatownlove/sendchinatownlove.github.io/blob/94478b8948af526e50f877a0061d55c0a7dd4ead/src/pages/VoucherRedemption/Amount.tsx#L149) and show it, but not have it be part of the value, or at restrict input to a-z only.

Just thinking about our data consistency, @justintmckibben, thoughts?